src/pages/components/Overlay.qml

```diff
@@ -158,6 +158,29 @@ PanelBackground {
         source: "image://theme/graphic-gradient-edge"
     }
 
+    Label {
+        id: privateModeLabel
```

Comment:
Actually above should be sibling of the Rectangle.... So ```anchors.fill: contentDimmer``` given that Rectangle at BrowserPage 123 has an id of contentDimmer.